packages/Card/src/Card.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Card.styles";
+import Footer from "./components/Footer";
+import Title from "./components/Title";
+import Metadata from "./components/Metadata";
+import Header from "./components/Header";
+import Text from "./components/Text";
+import Separator from "./components/Separator";
+import Content from "./components/Content";
+
+const propTypes = {
+  /** Body content of the card. */
+  children: PropTypes.node,
+
+  /** If the width of the card should span it's parent container (100%). */
+  isFullWidth: PropTypes.bool,
+
+  /** If the height of the card should span the height of its children elements. */
+  isAutoHeight: PropTypes.bool,
+
+  /** If the card is in an "active" or "selected" state. */
+  isActive: PropTypes.bool,
+
+  /** If the card is disabled. */
+  isDisabled: PropTypes.bool,
+
+  /** Callback to be executed when the card is clicked or activated by keyboard. Typically required. */
+  onClick: PropTypes.func,
+
+  /** Size of the card (font size, min-height, padding, etc). */
+  size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+};
+
+const defaultProps = {
+  children: null,
+  size: ShirtSizes.MEDIUM,
+  isFullWidth: false,
+  isAutoHeight: false,
+  isActive: false,
+  isDisabled: false,
+  onClick: () => {},
+};
+
+const Card = props => {
```

Comment:
please use `function Card(props) {}`

While arrow function will do the work, it's better for readability to use the `function Name() {}` to mark a point where the component starts.